share/picongpu/tests/PusherScaling/testScaling.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Aug 29 12:31:17 2023
+
+@author: hw
+"""
+
+from scipy.optimize import curve_fit
+import numpy as np
+import matplotlib.pyplot as plt
+
+def square(x, a, b, c):
+    return a*x**2 + b*x +c
+
+def cubic(x, a , b, c, d):
+    return a*x**3 + b*x**2 + c*x + d
+
+a = 1
+b = 0
+c = 0
+d = 0
+
+Xplot = np.linspace(0, 16)
+Yplot = cubic(Xplot, a, b, c, d)
+
+YSplot = square(Xplot, a, b, c)
+
+low,high = 0, 1e-7 # uniform distributuion [low, high)
+anz = 5
+
+x = (high - low) * np.random.random(anz) + low
+y = cubic(x, a, b, c, d)
+err = (1.1 - 0.9) * np.random.random(anz) + 0.9
+y = y * err
+
+x = np.array([1, 2, 4, 8 ,16])
+y = np.array([0.08605361848304671, 0.32843778731071427, 1.2953533235522245, 5.02259236909998, 18.95564097904385])
+
+quotients = np.zeros(4)
+for j in range(len(y)-1):
+    quotients[j] = y[j+1]/y[j]
+    
+print(quotients)
+print(np.mean(quotients))
+print(np.std(quotients))
+print((max(quotients)-min(quotients))/2)
+
+
+popt, pcov = curve_fit(square, x, y)
+popt3, pcov3 = curve_fit(cubic, x, y)
+a = popt[0]
+b = popt[1]
+c = popt[2]
+
+print(popt)
+
+chi_sq = np.sum((y-square(x, *popt))**2/square(x, *popt))
+chi_sq3 = np.sum((y-cubic(x, *popt3))**2/cubic(x, *popt3))
+print(chi_sq)
+print(chi_sq3)
+
+radii = [2.2885981, 2.28859706, 2.28859835, 2.28859962, 2.28860035, 2.28860038,
```

Comment:
What is this and how do you compute the values?

Reply:
this file was just for testing and is now removed